stores/moderationSubmissionsStore.ts

```diff
@@ -17,6 +17,22 @@ export const useModerationSubmissionsStore = defineStore(
         const selectedSubmissionId: Ref<string> = ref('')
         const selectedSubmissionData: Ref<Submission | undefined> = ref()
         const filteredSubmissionDataForListComponent: Ref<Submission[]> = ref([])
+        const updateSubmissionFromTopNav: Ref<boolean> = ref(false)
```

Comment:
adds states for manipulation between the ModEdit TopNav and the Form to submit the form

Reply:
when using a state store, you should keep your data abstracted from the usage of it. 
ex. instead of calling data `firstTabData` `secondTabData`, it should be `facilitiesData` `profData`. 

So in this case, we should remove TopNav and  make it clear what it's doing.